README.md

```diff
@@ -954,24 +954,51 @@ Keep in mind, queue operations and management is an advanced discipline. This st
 
 ### Database connections
 
-Each GoodJob execution thread requires its own database connection that is automatically checked out from Rails’ connection pool. For example:
+GoodJob job executor processes require the following database connections:
```

Comment:
"job executor process" is maybe just a fancy way to say "worker", but I saw "executor" a couple other times in the README and wasn't sure what the correct language for GoodJob was, so I just went with it.

Reply:
I'm trying to avoid using "worker"  because it can be  ambiguous: https://github.com/sidekiq/sidekiq/discussions/4971#discussioncomment-1216694

The meaning here is "any process where GoodJob is executing jobs (eg the executable or async-mode)". 

I agree with you that the current wording is clunky and not necessarily clearer.

Reply:
"job executor process" is kind of clunky, but it is at least less ambiguous than "worker," in my opinion. Mike Perham also refers to the same concept as a "process" in [the Sidekiq docs](https://github.com/sidekiq/sidekiq/wiki/Best-Practices#4-use-precise-terminology), but using that word alone is again ambiguous (a web server that is enqueuing jobs but not executing them is also running as a process), so sticking a qualifier on the front like "job executor" hopefully disambiguates the term.